File: OpenCredentialPublisher.Data/ViewModels/nG/PdfRequest.cs
using OpenCredentialPublisher.Data.Models.Enums;

namespace OpenCredentialPublisher.Data.ViewModels.nG
{
    public class PdfRequest
    {
        public PdfRequestTypeEnum RequestType { get; set; }
        public string LinkId{get; set;}
        public int? ClrId{get; set;}
        public string AssertionId {get; set;}
        public string EvidenceName {get; set;}
        public int? ArtifactId {get; set;}
        public string ArtifactName {get; set;}
        public bool CreateLink { get; set; }
    }
}

File: OpenCredentialPublisher.Wallet/src/ClientApp/src/app/shared/models/pdfRequest.ts
import { PdfRequestTypeEnum } from "./enums/pdfRequestTypeEnum";

export class PdfRequest {
  requestType: PdfRequestTypeEnum;
  linkId: string;
  clrId: number | null;
  assertionId: string;
  evidenceName: string;
  artifactId: number;
  artifactName: string;
  createLink: boolean;
}
